GeneralStateTests/VMTests/vmArithmeticTest/add.json

```diff
@@ -1,202 +1,265 @@
 {
-    "add" : {
-        "_info" : {
-            "comment" : "Ori Pomerantz qbzzt1@gmail.com",
-            "filling-rpc-server" : "evm version 1.10.0-unstable-498458b4-20210226",
-            "filling-tool-version" : "retesteth-0.1.0-accesslist+commit.4804afbb.Linux.g++",
-            "lllcversion" : "Version: 0.5.14-develop.2021.2.28+commit.9189ad7a.Linux.g++",
-            "source" : "src/GeneralStateTestsFiller/VMTests/vmArithmeticTest/addFiller.yml",
-            "sourceHash" : "6e5c09b96d22f188b9b8d146a827e5d96de14959e9e14c7af03eae550cd349cf",
-            "labels" : {
-                "0" : ":label add_neg1_neg1",
-                "1" : ":label add_neg1_4",
-                "2" : ":label add_neg1_1",
-                "3" : ":label add_0_0",
-                "4" : ":label add_1_neg1"
+    "": {
```

Comment:
I think I need to fix this; these tests are keyed under an apparently arbitrary name.